packages/macros/tests/babel/get-config.test.ts

```diff
@@ -146,6 +146,15 @@ describe(`getConfig`, function () {
           expect(code).toMatch(/doSomething\(undefined\)/);
         });
 
+        buildTimeTest(`does not collapse nullish coalescing for non embroider macros, nullish case`, () => {
+          let code = transform(`
+            const aKnownValue = {};
+            aKnownValue.foo = true;
+            result = aKnownValue?.foo;
+          `);
+          expect(code).toMatch(`result = aKnownValue`);
```

Comment:
Is it still ?.foo?

Reply:
There are 2 runs with the same test with and without preset env. So one converts it to manual null check, cannot make one version that matches both results